backend/Application/Aggregates/Contract/Jobs/_06_AddTokenAddress.cs

```diff
@@ -50,18 +51,20 @@ await Policies.GetTransientPolicy(GetUniqueIdentifier(), _logger, _options.Retry
             .ExecuteAsync(async () =>
             {
                 await using var context = await _contextFactory.CreateDbContextAsync(token);
+                var connection = context.Database.GetDbConnection();
                 var tokens = await context.Tokens
-                    .Where(t => (int)t.ContractIndex == identifier)
+                    .AsNoTracking()
+                    .Where(t => (int)t.ContractIndex == identifier && t.TokenAddress == null)
```

Comment:
Why "int" and not "long"?

Reply:
The identifier created from `Enumerable.Range` is of type `int`.